src/textual/widgets/_progress_bar.py

```diff
@@ -226,6 +228,9 @@ class ProgressBar(Widget, can_focus=False):
     gradient: reactive[Gradient | None] = reactive(None)
     """Optional gradient object (will replace CSS styling in bar)."""
 
+    BAR_RENDERABLE: BarRenderable = BarRenderable
+    """Optional BarRenderable class for custom ProgressBar icons."""
```

Comment:
Maybe change the docstring to something like:
```py
"""BarRenderable to use for rendering the bar-part of the ProgressBar"""
```
since it is not optional, it just has a default, and I'm not sure if 'icons' conveys the meaning.